lib/server_test.go

```diff
@@ -2881,8 +2881,8 @@ func TestPrometheusMetricsE2E(t *testing.T) {
 
 	gt.Expect(body).To(ContainSubstring(`# HELP api_request_count Number of requests made to an API`))
 	gt.Expect(body).To(ContainSubstring(`# TYPE api_request_count counter`))
-	gt.Expect(body).To(ContainSubstring(`api_request_count{api_name="enroll",ca_name="ca",status_code="201"} 1.0`))
-	gt.Expect(body).To(ContainSubstring(`api_request_count{api_name="enroll",ca_name="ca",status_code="401"} 1.0`))
+	gt.Expect(body).To(ContainSubstring(`api_request_count{api_name="enroll",ca_name="ca",status_code="201"} 1`))
```

Comment:
So I'm not sure if the change to using the promhttp handler ( https://github.com/hyperledger/fabric-ca/pull/157/files#diff-183232ffcd508f973f5c4867933ffabeR136 ) caused this or if I missed something elsewhere

Reply:
@sykesm  - does this make sense to you?   I'll dig a bit deeper, but just wondering if I broke something or if something in the underlying prom code changed when I updated the underlying package

Reply:
It makes sense: https://github.com/prometheus/common/commit/fdd3d500fc554db8fd97bbf83be49bbd8ad88304

Reply:
Also, we should probably try to improve the tests to make them less fragile. We have this issue with the metrics in fabric as well. These assertions were expedient but not necessarily "good".

Reply:
agreed on improving them ... just wanted to make sure I did not break anything this time